release-team.members.txt

```diff
@@ -4,6 +4,7 @@ gabrielwen@google.com
 jlewi@google.com
 kunming@google.com
 lunkai@google.com
+quanlin@google.com
```

Comment:
Can you put in alphabetical order please?

You also need to add yourself to the github org to get an invite to the github org.

Reply:
updated.